### DISCUSSION TITLE
Change color of svgImage at runtime

### DISCUSSION THREAD

**Discussion by bergsteinne** [ElinamLLC/SharpVectors#253] [Q&A]
Hello,

Is it possible to change the color of an svg in svgImage at runtime in wpf?

Best,

**Comment by paulushub** [comment]
For monochrome yes, see the SvgIcon samples in WpfTestResourceSvg.

**Comment by bergsteinne** [comment]
Thank you, one question regarding SvgIcon: In the samples you refer to, the source is set via **<svgc:SvgIcon.SvgSource>                           <![CDATA[**

Could I do the same and set the source of the SvgIcon to a svg file?

**Comment by paulushub** [comment]
> Could I do the same and set the source of the SvgIcon to a svg file?

Yes, it supports both SvgSource and UriSource properties similar to other controls.

**Comment by bergsteinne** [comment]
Thanks - sorry for the newbie question but how would the xaml def. be for UriSource to a local file using SvgIcon?

**Comment by paulushub** [comment]
In the same file you will see controls using "Source", this is the same as "UriSource". The "Source" is maintained for backward compatibility.
